web/src/hooks/getVenue.js

```diff
@@ -0,0 +1,32 @@
+import { useStaticQuery, graphql } from 'gatsby';
+
+export const getVenue = () => {
+  const { allSanityVenue } = useStaticQuery(
+    graphql`
+      query Venue {
+        allSanityVenue(filter: { active: { eq: true } }, limit: 1) {
+          edges {
+            node {
+              id
+              mapLink
+              name
+              description
+              address {
+                building
+                postcode
+                street
+                town
+              }
+              image {
+                asset {
+                  url
+                }
```

Comment:
Not sure if it's worth using Gatsby Image here? Got no particular preference since the image would be quite small anyway and it's not like it's a giant feature image for a blog. But if you were up for giving it a go then Sanity have their own set of image sharper fragments here:

https://www.gatsbyjs.org/packages/gatsby-source-sanity/#using-images

Reply:
Good point, will take a lot at the docs and add it in.